src/components/application_manager/src/commands/command_request_impl.cc

```diff
@@ -298,7 +298,7 @@ void CommandRequestImpl::SendResponse(
     response[strings::msg_params] = *response_params;
   }
 
-  if (info) {
+  if (info && !info.empty()) {
```

Comment:
Does not compile since its a char array, need to check if first character is '\0'

```suggestion
  if (info && *info != '\0') {
```